doc/users/credits.rst

```diff
@@ -231,100 +402,171 @@ Martin Spacek,
 Martin Teichmann,
 Martin Thoma,
 Martin Ueding,
+Massimo Santini,
 Masud Rahman,
 Mathieu Duponchelle,
 Matt Giuca,
+Matt Hancock,
 Matt Klein,
 Matt Li,
 Matt Newville,
 Matt Shen,
 Matt Terry,
+Matthew Bell,
 Matthew Brett,
 Matthew Emmett,
+Matthew Newville,
 Matthias Bussonnier,
+Matthias Lüthi,
 Matthieu Caneill,
+Matti Picus,
 Matěj Týč,
 Maximilian Albert,
+Maximilian Maahn,
+Maximilian Nöthe,
 Maximilian Trescher,
 Mellissa Cross,
+Mher Kazandjian,
 Michael,
 Michael Droettboom,
 Michael Sarahan,
+Michael Scott Cuthbert,
+Michael Seifert,
 Michael Welter,
 Michiel de Hoon,
 Michka Popoff,
+Mike Henninger,
+Mike Jarvis,
 Mike Kaufman,
 Mikhail Korobov,
 MinRK,
 Minty Zhang,
 MirandaXM,
 Miriam Sierig,
+Mitar,
+Molly Rossow,
+Moritz Boehle,
+Mudit Surana,
 Muhammad Mehdi,
+Naoya Kanai,
+Nathan Goldbaum,
+Nathan Musoke,
+Nathaniel M. Beaver,
 Neil,
 Neil Crighton,
 Nelle Varoquaux,
 Niall Robinson,
 Nic Eggert,
 Nicholas Devenish,
+Nick Garvey,
+Nick Papior,
 Nick Semenkovich,
+Nico Schlömer,
 Nicolas P. Rougier,
 Nicolas Pinto,
+Nicolas Tessore,
+Nik Quibin,
 Nikita Kniazev,
 Niklas Koep,
 Nikolay Vyahhi,
+Nils Werner,
+Ninad Bhat,
 Norbert Nemec,
+Norman Fomferra,
 OceanWolf,
 Oleg Selivanov,
 Olga Botvinnik,
 Oliver Willekens,
+Olivier,
+Orso Meneghini,
+Osarumwense,
+Pankaj Pandey,
 Parfenov Sergey,
 Pascal Bugnion,
+Pastafarianist,
 Patrick Chen,
 Patrick Marsh,
 Paul,
 Paul Barret,
 Paul G,
+Paul Ganssle,
```

Comment:
Paul G (just above) is Paul Ganssle, please add an entry to the mailmap (see e.g. git help shortlog).